Bottom: a34badf15780633755bb426f23f7e1cf195eb9de
Top:    98e44c728c4aa84e78e984692439dfbcf81abebd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 14:55:46 +0100

Refresh of bentley-ottann-test-implementa

---

diff --git a/src/sweep.h b/src/sweep.h
index 257664f..3af86b8 100644
--- a/src/sweep.h
+++ b/src/sweep.h
@@ -1,4 +1,4 @@
-#include "cairo/cairo-traps-private.h"
+#include "borast/cairo-traps-private.h"
 
 cairo_status_t bo_poly_to_traps (POLYAREA *poly, cairo_traps_t *traps);
 cairo_status_t bo_contour_to_traps (PLINE *contour, cairo_traps_t *traps);
